cirq-core/cirq/transformers/noise_adding.py

```diff
@@ -0,0 +1,62 @@
+from collections.abc import Mapping
+
+import cirq
+import numpy as np
+
+
```

Comment:
transformers need to be annotated and to implement the transformer API
```suggestion
@transformer_api.transformer
```

Reply:
Do you know how to fix the errors that I'm getting now that I have done this?

Reply:
@NoureldinYosri It's still failing. Do you know how to fix it?

Reply:
this is an argument order issue .. you can use the following but you need to set a default value for `p`
```suggestion
@transformer_api.transformer
def add_depolarizing_noise_to_two_qubit_gates(
    circuit: circuits.AbstractCircuit,
    context: transformer_api.TransformerContext | None = None,
    p: float | Mapping[tuple[ops.Qid, ops.Qid], float] = 0.5,  # some default value
    target_gate: ops.Gate = ops.CZ,
    rng: np.random.Generator | None = None,
) -> circuits.AbstractCircuit:
```